tpgtools/overrides/containerazure/samples/nodepool/basic_update.tf.tmpl

```diff
@@ -80,7 +85,7 @@ resource "google_container_azure_node_pool" "primary" {
 
   name      = "{{node-pool-name}}"
   subnet_id = "/subscriptions/{{azure_subscription}}/resourceGroups/{{byo_multicloud_prefix}}-dev-byo/providers/Microsoft.Network/virtualNetworks/{{byo_multicloud_prefix}}-dev-vnet/subnets/default"
-  version   = "1.21.6-gke.1500"
+  version   = "${data.google_container_azure_versions.versions.valid_versions[0]}"
```

Comment:
Are these handwritten? This will make it significantly more difficult to change this to be based off the samples in the DCL repo.